stuff/profiles/layouts/settings/commandbar.xml

```diff
@@ -25,6 +25,9 @@
     <command>MI_EnterGroup</command>
     <command>MI_ExitGroup</command>
     <separator/>
+    <command>MI_VectorGuidedDrawing</command>
```

Comment:
In my opinion we don't need this as the default, these are very advanced features and sadly they are undocumented. I've tried to create documentation myself, but there are many shortcomings outside of that, that I feel it's too advanced and cumbersome to present to general users, the feature is unfinished. There's many prerequisites the user has to do before learning how to use these. I suggest leaving these out for now. This goes for the Drawing Controls panel as well (the dockables don't need to be commands on the panel).